miso-web/src/spring-test/java/uk/ac/bbsrc/tgac/miso/webapp/springtest/AbstractST.java

```diff
@@ -223,10 +224,15 @@ private String pollingResponserHelper(String requestType, List<?> dtos, String c
             .andReturn();
         break;
     }
-
-    String id = JsonPath.read(mvcResult.getResponse().getContentAsString(), "$.operationId");
-    String response = pollingResponse(controllerBase + "/bulk/" + id);
-    return response;
+    try {
+      String id = JsonPath.read(mvcResult.getResponse().getContentAsString(), "$.operationId");
+      String response = pollingResponse(controllerBase + "/bulk/" + id);
+      return response;
+    } catch (IllegalArgumentException e) {
+      // this will happen if no content (HTTP 204) is returned
```

Comment:
Provided 204-handling is kept in this method: this might also catch and silently pass actual issues. Test the status early to decide what to do instead of trying and catching the exception caused by doing the incorrect thing for this case.